README.md

```diff
@@ -260,72 +293,68 @@ ID                         NAME                  REQUIRED CREDENTIALS
 54b7d6b5d8a5974f2c000002   Microsoft Azure       [subscription_id cert_management_certificate]          [server]
 5579b79bd8a5978602000002   Softlayer             [username api_key]                                     [server]
 ```
-Take also into account that you should have configured your credentials before, using the Web UI or concerto settings cloud_accounts create. We will choose Digital Ocean, whose ID is `53f0f09ad8a5975a1c000010`.
 
+Take also into account that you should have configured your credentials before, using the Web UI or `concerto settings cloud_accounts create`. We will choose Digital Ocean, whose ID is `53f0f09ad8a5975a1c000010`.
 
-The server's FQDN will be formed using one of your domains, and a given hostname. You can bring your own domains using Concerto Web UI or CLI command concerto dns_domains subcommands.
-```
-$ concerto dns_domains list
-ID NAME TTL CONTACT MINIMUM ENABLED
-5601726ffef51ac134000028 concertointernal.concerto.io 10800 ns@concerto.io 10800 true
-```
-We have only the default domain. Take note of its ID, `5601726ffef51ac134000028`, and choose a hostname suffix for the new server.
+***TODO*** The server's FQDN will be formed using one of your domains, and a given hostname; Choose a hostname suffix for the new server.
```

Comment:
Remove reference about FQDN: with the removal of the managed DNS feature this is no longer needed